### PR TITLE
fix: structured data co-hosts and show image

### LINKED ISSUE #12: Speakers page image and Main logo is not shown
The following images are not shown:
- Sidebar logo
- Speaker picture inside speaker page

### PR DESCRIPTION
**Pull request by JellyBellyDev**
closed #12

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -163,3 +163,49 @@ export const markdown = (mdText:string)=> {
...
+        "name": "Andrea Giannantonio",
+        "nickName": "JellyBellyDev",
+        "about": "Abruzzese DOC ma romano d’adozione, grande appassionato di tecnologia, musica rap, serie tv e buon cibo. Si laurea in Tecnologie Informatiche alla Sapienza di Roma e dal 2008 macina codice in php e js con tanto focus sull'automazione. La sua fame di sapere lo porta ad esplorare sempre nuove tecnologie e metodologie. Negli ultimi anni ha sposato a pieno il tema della developer experience ed è un attivo sostenitore di diverse community e del mondo open source.",
+        "imageURL": "/JellyBellyDev.png"
+      }
+  ]}
+
+  return coH.coHosts;
+
+};
```
Comment (brainrepo) [inline src/utils.ts]: NIT: Why are you exporting a function and not directly a constant?

```

const CO_HOST = [{
        "name": "mattia tommasone",
        "nickName": "raibaz",
        "about": "Software developer focused on business logic, web applications and 3d graphics, but with a broad spectrum of tiny projects realized for myself and friends up to now",
       ...
}, ....
]
```

**Review by brainrepo — COMMENTED** [review]
Thank you,
Very great work! 
I liked the idea to extracts thecohost in a utils file.
Just one little nit, happy to approve once changed

File: src/utils.ts (hunk 1 above)
Comment (JellyBellyDev) [reply src/utils.ts]: You are a wise man! ;)  
I refactored!

**Comment by JellyBellyDev** [comment]
@brainrepo maybe we should rename the directory path from `/speakers/nickname` to `/hosts/nickname`?

and shouldn't we also add a link to your image leading to your bio?

**Comment by brainrepo** [comment]
> @brainrepo maybe we should rename the directory path from `/speakers/nickname` to `/hosts/nickname`?
> 
> and shouldn't we also add a link to your image leading to your bio?

Yep good point. I merge this and we create a new issue

**Pull request merged**